src/modules/commander/HomePosition.hpp

```diff
@@ -45,6 +45,8 @@ static constexpr int kHomePositionGPSRequiredFixType = 2;
 static constexpr float kHomePositionGPSRequiredEPH = 5.f;
 static constexpr float kHomePositionGPSRequiredEPV = 10.f;
 static constexpr float kHomePositionGPSRequiredEVH = 1.f;
+static constexpr float kMinHomePositionChangeEPH = 1.f;
+static constexpr float kMinHomePositionChangeEPV = 1.5f;
```

Comment:
Why 1.5?

Reply:
Just from some testing i had more than 1m variation. In the altitude we can have a larger error imo, as for landing we only have to make sure, the vehicle is at the final landing speed, when we do not have a distance sensor. This starts at MPC_LAND_ALT2 which s per default at 5m. SO if we have an initial error of 1.5m at start and while landing the error in the opposite direction, we still should be fine (but i guess we should not set it higher than 2m or make it dependend on MPC_LAND_ALT2).